tests/cypress/support/commands.js

```diff
@@ -56,8 +57,11 @@ Cypress.Commands.add('openTaskJob', (taskName) => {
 })
 
 Cypress.Commands.add('createShape', (ferstX, ferstY, lastX, lastY) => {
-    cy.get(':nth-child(8) > svg').trigger('mousemove').click()
-    cy.get(':nth-child(6) > :nth-child(1) > .ant-btn').click()
+    cy.get('.cvat-draw-rectangle-control').click()
+    cy.get('.cvat-draw-shape-popover-content')
+    .find('button')
+    .contains('Shape')
+    .click({force: true})
```

Comment:
What does the argument ``force: true`` do?

Reply:
Forcing a click overrides the actionable checks Cypress applies and will automatically fire the events.
From Cypress log (without forcing):
```
Timed out retrying: cy.click() failed because this element:

<span>Shape</span>

has CSS pointer-events: none

pointer-events: none prevents user mouse interaction.

Fix this problem, or use {force: true} to disable error checking.
```